output.md

```diff
@@ -90,59 +95,73 @@ file.h5
 2.  The name of the HDF5 file (e.g. `fields.h5`) where the coefficients of all FEM fields will be written.
 
-References to the available FEM spaces are given to the OutputManager object through the `add_spaces(**kwargs)` method, and the corresponding YAML file is created upon calling the method `export_spaces_info()`. In order to inform the OutputManager object that the next fields to be exported are time-independent, the user should call the `set_static()` method. In the case of time-dependent fields, the user should prepare a time snapshot (which is defined for a specific time step index `ts` and time value `t`) by calling the method `add_snapshot(t, ts)`. In both cases the fields are exported to the HDF5 file through a call to the method `export_fields(**kwargs)`. Here is a usage example:
+References to the available FEM spaces are given to the OutputManager object through the `add_spaces(**kwargs)` method, and the corresponding YAML file is created upon calling the method `export_spaces_info()`. In order to inform the OutputManager object that the next fields to be exported are time-independent, the user should call the `set_static()` method. In the case of time-dependent fields, the user should prepare a time snapshot (which is defined for a specific time step `ts` and time value `t`) by calling the method `add_snapshot(t, ts)`. In both cases the fields are exported to the HDF5 file through a call to the method `export_fields(**kwargs)`. Here is a usage example:
 
 ```python
 # SymPDE Layer
-# Discretization 
+# Discretization
 # V0h and V1h are discretized SymPDE Space
 # u0 and u1 are FemFields belonging to either of those spaces
 output_m = OutputManager('spaces.yml', 'fields.h5')
 
-output_m.add_spaces(V0=V0h, V1=V1h) 
+output_m.add_spaces(V0=V0h, V1=V1h)
+output_m.export_spaces_info() # Writes the space information to Yaml
 
 output_m.set_static() # Tells the object to save in /static/
 output_m.export_fields(u0_static=u0, u1_static=u1) # Actually does the saving
 
-output_m.add_snapshot(t=0., ts=0) 
+output_m.add_snapshot(t=0., ts=0)
 # The line above tells the object to:
 # 1. create the group snapshot_x with attribute t and ts
 # 2. save in this snapshot
 output_m.export_fields(u0=u0, u1=u1)
-
-output_m.export_spaces_info() # Writes the space information to Yaml
 ```
 
 ## Usage of class `PostProcessManager`
 
 Typically the `PostProcessManager` class is used in a separate post-processing script, which is run after the simulation has finished. In essence it evaluates the FEM fields over a uniform grid (applying the appropriate push-forward operations) and exports the values to a VTK file (or a sequence of files in the case of a time series). An instance of the `PostProcessManager` class is created by specifying the following:
 
-1.  The name of the geometry file (in HDF5 format) which defines the multi-patch geometry of interest
+1.  The name of the geometry file or the topological domain
```

Comment:
I think this point needs a bit more explanation. Could you not keep (some of) the original text?

Reply:
Put back some of the text in [34bfd98](https://github.com/pyccel/psydac/pull/234/commits/34bfd9845a7a7c44d8589ff9b670b919f204d457)